pkg/kube/multicluster/secretcontroller.go

```diff
@@ -126,13 +131,21 @@ func (c *Controller) AddHandler(h ClusterHandler) {
 	c.handlers = append(c.handlers, h)
 }
 
+func (c *Controller) GetClusterStore() *ClusterStore {
+	return c.cs
+}
+
 // Run starts the controller until it receives a message over stopCh
 func (c *Controller) Run(stopCh <-chan struct{}) error {
 	// run handlers for the local cluster; do not store this *Cluster in the ClusterStore or give it a SyncTimeout
```

Comment:
Keep in mind the "local cluster" isn't the cluster housing an external istiod. Instead, it's the "config cluster". External istiod is ~mostly designed as if istiod is running outside _any_ cluster.

Reply:
Big +1, `local` already used to indicate the configcluster

Reply:
In this pr, the `KUBECONFIG` env is used and has a totally different meaning with `kubeconfig`, that's not align with developer concept.